icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc

```diff
@@ -1117,31 +1118,21 @@ sbn::crt::CRTHit CRTHitRecoAlg::MakeSideHit(vector<art::Ptr<CRTData>> coinData,
     t1trigs.resize(std::distance(t1trigs.begin(), std::unique(t1trigs.begin(), t1trigs.end())));
     //    t1hit = std::reduce(t1trigs.begin(), t1trigs.end()) / (uint64_t)t1trigs.size();
     //    t1hit = std::accumulate(t1trigs.begin(), t1trigs.end()) / (uint64_t)t1trigs.size();
-    if(ttrigs.size()>10){
+    if(!ttrigs.empty()){
+      uint64_t const offset = ttrigs[0];
+      int64_t rel_thit = 0;
       for(uint64_t const t : ttrigs){
-        thit_s = t - t%1'000'000'000;
-	if (region=="North" || region=="South") thit_ns += t%1'000'000'000-uint64_t(200.*fPropDelay)-fSiPMtoFEBdelay;
-	else thit_ns += t%1'000'000'000 -uint64_t(400.*fPropDelay)-fSiPMtoFEBdelay; 
+        rel_thit += t - offset;
+	thit -= RegionDelay(region);
```

Comment:
Should this be `rel_thit`?
```suggestion
	rel_thit -= RegionDelay(region);
```

Reply:
ah yes, thank you @PetrilloAtWork!